composer.json

```diff
@@ -13,6 +13,7 @@
         "monolog/monolog": "^1.17|^2.0",
         "phpseclib/phpseclib": "~2.0||^3.0.2",
         "guzzlehttp/guzzle": "~5.3.3||~6.0||~7.0",
+        "symfony/polyfill-php81": "^1.23.0",
```

Comment:
`symfony/polyfill-php81`  provides `ReturnTypeWillChange`, otherwise using the attribute will trigger an error in PHP < 8.1. 
See: https://github.com/symfony/polyfill-php81/blob/main/Resources/stubs/ReturnTypeWillChange.php

Reply:
From my experience it isn't necessary, without this polyfill using the new attribute on PHP 8.0 doesn't trigger any errors, you can see this from green PHP 8.0 builds here:
* https://github.com/phpspec/prophecy/pull/533
* https://github.com/php-amqplib/php-amqplib/pull/929
* https://github.com/Behat/Behat/pull/1355

Before PHP 8.0 attributes are treated as just a line with a comment.

See also https://php.watch/versions/8.0/attributes
> It is optional to match the Attribute name to a class name.
> If an attribute does not map to a class, the Reflection API does not allow to fetch an instance of the instantiated attribute.